jujugui/static/gui/src/app/app.js

```diff
@@ -1959,11 +1959,12 @@ YUI.add('juju-gui', function(Y) {
       @param {Function} next - Call to continue dispatching.
     */
     _renderStatusView: function(state, next) {
+      const propTypes = window.juju.components.Status.propTypes;
       ReactDOM.render(
         <window.juju.components.Status
-          db={this.db}
-          model={this.env.getAttrs()}
-          urllib={window.jujulib.URL}
+          db={shapeup.fromShape(this.db, propTypes.db)}
```

Comment:
Could you define `fromShape` as a top level variable in this file so that we can reduce some of the typing :)